backend/medtagger/repositories/users.py

```diff
@@ -1,6 +1,9 @@
 """Module responsible for definition of Users' Repository."""
 from typing import List, Optional
 
+from sqlalchemy.orm.exc import NoResultFound
+
+from medtagger.api import InvalidArgumentsException
```

Comment:
Repositories can be used outside of an API context, so it shouldn't import such Exception.